db/seeds.rb

```diff
@@ -5,5 +5,21 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-puts "Importing Schools information, this may take a minute..."
-SchoolDataImporter.new.run
+
+unless Rails.env.development?
+  raise "This db/seeds.rb file is only intended for use in a local development environment. It should not be used in the #{Rails.env} environment."
```

Comment:
I don't think we should raise an error here. The seeds file is called as part of `db:setup` so could be called when setting up a production environment. We could just wrap the creation of the fixtures/factories in a conditional?

Reply:
Or we could change our Docker entrypoint to just run `db:create db:schema:load`

Reply:
I think you're right; the seeds can in theory be used to seed a production database as well. I think the best thing would be to just wrap the code in a conditional.